jpeg-8c/makefile.vc

```diff
@@ -206,6 +206,23 @@ setup-v15:
 	ren makewvcx.v15 wrjpgcom.vcxproj
 	ren makewfil.v15 wrjpgcom.vcxproj.filters
 
+setup-v17:
+	ren jconfig.vc jconfig.h
+	ren makejsln.v17 jpeg.sln
+	ren makeasln.v17 apps.sln
+	ren makejvcx.v17 jpeg.vcxproj
+	ren makejfil.v17 jpeg.vcxproj.filters
+	ren makecvcx.v17 cjpeg.vcxproj
+	ren makecfil.v17 cjpeg.vcxproj.filters
+	ren makedvcx.v17 djpeg.vcxproj
+	ren makedfil.v17 djpeg.vcxproj.filters
+	ren maketvcx.v17 jpegtran.vcxproj
+	ren maketfil.v17 jpegtran.vcxproj.filters
+	ren makervcx.v17 rdjpgcom.vcxproj
+	ren makerfil.v17 rdjpgcom.vcxproj.filters
+	ren makewvcx.v17 wrjpgcom.vcxproj
+	ren makewfil.v17 wrjpgcom.vcxproj.filters
```

Comment:
Bleah. I realize you didn't invent this convention, but it makes me sad to perpetuate it.

Reply:
I'm somewhat surprised this is necessary. I've been able to build older vcxproj/solutions using VS2022 as long as the PlatformToolset version is set correctly.

Reply:
i suppose i couldve just made an empty target but i figured id follow convention in case more was needed. probably a lot of this could be cleaned up